stargate_network/index.md

```diff
@@ -4,68 +4,58 @@ nav_order: 70
 has_children: true
 ---
 
-{: .note }
-This content was migrated from the old wiki and is awaiting an update.
-
 1. Table of Contents
 {:toc}
 
 # Stargate Network
+Stargate Network is a complex system that allows Stargates to communicate and establish wormhole connections between each other. Each Stargate has up to three specific addresses that allow other Stargates to dial itself. Stargate Network also is responsible for choosing the optimal Stargate whenever a Solar System is dialed.
 
-# Addresses
-
-There are 3 kinds of Addresses:
-* `7-chevron Addresses` can be used to dial a Solar System inside the same galaxy. If a Solar System is a part of multiple Galaxies, it can have a different 7-chevron Address in each Galaxy.
-* `8-chevron Addresses` can be used to dial any Solar System. Each Solar System has only one 8-chevron Address.
-* `9-chevron Addresses` can be used to dial specific Stargates. Each Stargate has a unique 9-chevron Address, which can be used to dial it anywhere, even when it's moved to another Solar System where its 7 and 8-chevron Address changes.
+## Addresses
+There are 3 kinds of addresses the Stargates may have. Each of them serve different purposes, commonly distinguished by their length:
+* `7-Symbol address`, also called "Solar System address", is an address used to dial a particular Solar System. This means the Stargate Network will pick the most suitable Stargate to route the incoming connection within the Solar System. Keep in mind that 7-Symbol addresses work only inside of the same galaxy, with only exception being if the given Solar System is a part of two Galaxies simultaneously.
```

Comment:
Please do not mix chevron vs symbol address length.

Reffer to them as 7-chevron, 8-chevron and 9-chevron addresses.

You can make a note there, that someone can call them 6, 7 and 8 symbol address, but wiki will consistently always use **chevrons**.